vita3k/gui/src/app_context_menu.cpp

```diff
@@ -218,8 +218,8 @@ void delete_app(GuiState &gui, EmuEnvState &emuenv, const std::string &app_path)
         const auto PREF_PATH = fs::path(emuenv.pref_path);
         fs::remove_all(PREF_PATH / "ux0/app" / app_path);
 
-        const auto BASE_PATH = fs::path(emuenv.base_path);
-        const auto CUSTOM_CONFIG_PATH{ BASE_PATH / "config" / fmt::format("config_{}.xml", app_path) };
+        const auto CACHE_PATH = fs::path(emuenv.cache_path);
+        const auto CUSTOM_CONFIG_PATH{ PREF_PATH / "config" / fmt::format("config_{}.xml", app_path) };
```

Comment:
Revert all you change relate set pref path everywhere!
Pref path only contain psvita folder ux0, etc... no any other things
Custom config, config and all other is inside base path is Only inside base path

Reply:
On some OS, base path can be read only, so custom config cannot be inside base path in this case.

Reply:
OK but change nothing here, i said revert for said no using pref path 🤷‍♂️

Reply:
there is no pref path in the code you commented.

Reply:
> there is no pref path in the code you commented.

Euuu ? Is it ? Is litteraly writed Pref path is have coded 🗿

Reply:
Ah sorry indeed, my bad. BASE_PATH must be changed to config_path and not pref_path then. (Although between those 2 it's more a question of preference)

Reply:
Yes, but just using config_path for set custom config is good
Like i said, pref path is reserved for psvita folder